inputs-required.tf

```diff
@@ -0,0 +1,24 @@
+variable "resource_group_name" {
+  description = "Enter name of Azure resource group"
```

Comment:
we don't need to put enter at the start of each sentence it looks bad for the docs, see an example at https://github.com/hmcts/terraform-module-postgresql-flexible#inputs